ci/owncloud/upload_to_owncloud.sh

```diff
@@ -28,8 +28,24 @@ usage() {
     echo ""
 }
 
+create_dirs() {
+    # Recursively create directory path in remote
+    # $1 remote path to create
+    local path=
+    for dir in $(echo "$1" | tr "/" "\\n"); do
```

Comment:
Doesn't shellcheck complain about this?

I think a more typical pattern would be
```suggestion
    echo "$1" | tr / '\n' | while read -r -s dir; do
```

Reply:
IMO `for` loop is easier to read, why is while preferred?

Reply:
Actually @rmelotte 's suggestion to use `IFS` is the best IMO, so changed to that one.